Libplanet.Tests/Store/StoreTest.cs

```diff
@@ -1059,6 +1059,68 @@ public void Copy()
             }
         }
 
+        [Fact]
```

Comment:
```suggestion
        [SkippableFact]
```

Tests seem to fail in `Windows_Mono`, `Windows_NETFramework` because of this line. Doesn't it need `SkippableFact`? 

> Error message
Libplanet.RocksDBStore.Tests.RocksDBStoreTest.PruneBlockStates [FAIL]
      Xunit.SkipException : RocksDB is not available.
      Stack Trace:
          at Libplanet.RocksDBStore.Tests.RocksDBStoreTest..ctor () [0x00027] in <47a371ce115c49408742f0c17cb5fc8d>:0 
          at (wrapper managed-to-native) System.Reflection.RuntimeConstructorInfo.InternalInvoke(System.Reflection.RuntimeConstructorInfo,object,object[],System.Exception&)
          at System.Reflection.RuntimeConstructorInfo.InternalInvoke (System.Object obj, System.Object[] parameters, System.Boolean wrapExceptions) [0x00005] in <06aa38aacf3b452693a1e023c55f36bf>:0